src/features/customerList/tableTools/CustomerListTableTools.tsx

```diff
@@ -69,7 +65,11 @@ const CustomerListTableTools = <T extends string>({
           onChange={(e) => setSearchVal((e.target as HTMLInputElement).value)}
         />
       </div>
-      <Modal isOpen={messageModalOpen} toggleModal={() => setMessageModalOpen(false)}>
+      <Modal
+        label={t('customerList.message.header').toUpperCase()}
```

Comment:
Now that the title is shown with the spinner the UX is a bit worse I think. IMO looked better when the modal was shown with the spinner only :)

Reply:
Indeed, fixed 👍